cli/internal/cmd/apply.go

```diff
@@ -40,6 +40,32 @@ import (
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+// NewApplyCmd creates the apply command.
+func NewApplyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apply",
+		Short: "Apply a configuration to a Constellation cluster",
+		Long:  "Apply an upgrade to a Constellation cluster by applying the chosen configuration.",
```

Comment:
I think that `Apply an upgrade` might be a bit misleading. Maybe somehting like `Apply the current configuration. This can be used to initialize or upgrade the Constellation.`